avro/src/main/java/com/fasterxml/jackson/dataformat/avro/jsr310/AvroJavaTimeModule.java

```diff
@@ -0,0 +1,61 @@
+package com.fasterxml.jackson.dataformat.avro.jsr310;
+
+import com.fasterxml.jackson.core.Version;
+import com.fasterxml.jackson.core.json.PackageVersion;
```

Comment:
Oh no, do not use this -- that would give module name of `jackson-core` :)

Instead you will want to use some of `com.fasterxml.jackson.dataformat.avro.PackageVersion` except... I'll add another note on details.

Reply:
Right `com.fasterxml.jackson.dataformat.avro.PackageVersion` used instead of one from `jackson.core`.